### PR TITLE
Allow prebid for 300x600 and 970x250 ad sizes

### PR DESCRIPTION
**Pull request by regiskuckaertz**
The US is now confident they can user header bidding for all slot sizes. This PR is prepared ahead of time and will be deployed as soon as all the line items have been set up correctly in DFP.

/cc @jbreckmckye

### CODE REVIEW

**Comment by jbreckmckye** [comment]
Looks good to me 👍

File: static/src/javascripts/projects/common/modules/commercial/dfp/PrebidService.js [hunk 1]
```diff
@@ -87,14 +87,13 @@ define([
...
 
-    function getMatchingTrialSizes(advert) {
-        // For the purpose of the US trial, we will only be bidding for leaderboard ads (728x90) and single-size MPUs (300x250)
-        var trialSizes = [[728,90], [300,250]];
+    function getMatchingSizes(advert) {
+        var sizes = [[728,90], [300,250], [300,600], [970,250]];
         var advertSizes = concatAll(values(advert.sizes));
 
-        return trialSizes.filter(function (trialSize) {
+        return sizes.filter(function (size) {
```
Comment (uplne) [inline static/src/javascripts/projects/common/modules/commercial/dfp/PrebidService.js]: Are you sure about this?

Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/commercial/dfp/PrebidService.js]: huhuhu

**Comment by jbreckmckye** [comment]
Is there anything stopping us from merging this? The US team have been asking after it.

**Comment by uplne** [comment]
Yeah 🚢

**Comment by regiskuckaertz** [comment]
The line items have not been configured with the new sizes yet. We're on it 😄

**Pull request merged**